index.js

```diff
@@ -13,7 +13,7 @@ var crypto = require("crypto");
 var sjcl = require("sjcl");
 var uuid = require("uuid");
 var validator = require("validator");
-var privateToAddress = require("ethereumjs-util").privateToAddress;
+var secp256k1 = require("secp256k1/elliptic");
```

Comment:
@tinybike this [elliptic](https://github.com/indutny/elliptic) wrapper for nice interface